k8/k8-mongo-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mongodb
+spec:
+  type: LoadBalancer
+  selector:
+    app: mongodb
+  ports:
+  # List the API port first so that NEO4J_SERVICE_PORT contains this value.
+  - protocol: TCP
+    name: "mongodb-api"
```

Comment:
Detail. Apparently, if a service exports only one port (as is this case), one need not give it. a name. I.e., this line can be dropped. 

Then, the MONGODB_SERVICE_PORT is defined as this port and no extra env. variable MONGODB_SERIVICE_PORT_MONGODB_API port is defined .....